src/lib/style/components/_markdown.scss

```diff
@@ -116,25 +116,25 @@
 }
 .markdown p {
   margin: 0px;
-  padding-bottom: 12px;
+  margin-bottom: 12px;
```

Comment:
Typography styles should use top/bottom margin, not padding, because in a typographic system margin collapse is intentional (and looks weird when it doesn't).